### PR TITLE
Update placeholder text color

### PR DESCRIPTION
**Pull request by iamthuypham**

### CODE REVIEW

**Comment by iamthuypham** [comment]
Waiting for Amber to double check and approve the change.

File: components/shared-styles/index.js [hunk 1]
```diff
@@ -95,6 +95,7 @@ export const inputColors = {
 	inputBorderColor: colors.borderColor,
 	inputFocusedBorderColor: '#278ed4',
 	inputFocusedShadowColor: '#d0e6f6',
+	inputPlaceholderColor: '#C7C7C7',
```
Comment (bryanrsmith) [inline components/shared-styles/index.js]: We shouldn't duplicate this in two theming systems. Shared styles will eventually be removed, so it'd be great to move away from it and use theme instead.

**Comment by iamthuypham** [comment]
We approved the #c7c7c7 here : https://faithlife.com/messages/24703?threadId=801163

**Comment by iamthuypham** [comment]
@bryanrsmith I removed the `inputColors` in select.jsx

**Pull request merged**